### PR TITLE
Add 'Learnship Networks GmbH' (community contribution)

### PR DESCRIPTION
**Pull request by datenanfragen-community-edits**
This suggestion was submitted through the website.

**[Edit](https://company-json.netlify.com/#!doc=%7B%22slug%22%3A%22learnship%22%2C%22relevant-countries%22%3A%5B%22all%22%5D%2C%22categories%22%3A%5B%22school%22%5D%2C%22name%22%3A%22Learnship%20Networks%20GmbH%22%2C%22address%22%3A%22Stolberger%20Str.%20374%5CnD-50933%20K%C3%B6ln%22%2C%22phone%22%3A%22%2B49%20221%2013062820%22%2C%22email%22%3A%22support%40learnship.de%22%2C%22web%22%3A%22https%3A%2F%2Fwww.learnship.com%22%2C%22sources%22%3A%5B%22https%3A%2F%2Fwww.learnship.com%22%2C%22https%3A%2F%2Fwww.northdata.com%2FLearnship%2BNetworks%2BGmbH%2C%2BK%25C3%25B6ln%2FHRB%2B61591%22%2C%22https%3A%2F%2Fwww.linkedin.com%2Fcompany%2Flearnship%2F%22%2C%22https%3A%2F%2Fgithub.com%2Fdatenanfragen%2Fdata%2Fpull%2F1034%22%5D%2C%22suggested-transport-medium%22%3A%22email%22%2C%22quality%22%3A%22verified%22%7D)**

### CODE REVIEW

File: companies/learnship.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "school"
+    ],
+    "name": "Learnship Networks GmbH",
+    "address": "Stolberger Str. 374\nD-50933 Köln",
+    "phone": "+49 221 13062820",
+    "email": "support@learnship.de",
```
Comment (fm) [inline companies/learnship.json:12]: DPO email is actually `DPO@learnship.com`

File: companies/learnship.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "web": "https://www.learnship.com",
+    "sources": [
+        "https://www.learnship.com",
```
Comment (fm) [inline companies/learnship.json:15]: I would replace with https://www.learnship.com/en/european-privacy-policy/

File: companies/learnship.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    "slug": "learnship",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "school"
+    ],
+    "name": "Learnship Networks GmbH",
+    "address": "Stolberger Str. 374\nD-50933 Köln",
+    "phone": "+49 221 13062820",
```
Comment (fm) [inline companies/learnship.json:11]: Phone should be +49 221 66954960

**Review by fm — CHANGES_REQUESTED** [review]

**Review by baltpeter — APPROVED** [review]
Thank you!

**Pull request merged**